__tests__/pages/profile/username.test.js

```diff
@@ -57,7 +57,7 @@ describe('user profile test', () => {
     const { container, findByRole, queryByText } = render(tree)
     await findByRole('heading', { name: /loading/i })
     await waitForElementToBeRemoved(() => queryByText('Loading...'))
-    expect(container).toMatchSnapshot()
+    // expect(container).toMatchSnapshot()
```

Comment:
You forgot to remove this comment.

Reply:
Ah, I thought you wanted this removed because it didn't need a snapshot. Do I submit a separate pull request for this file that removes this line?